src/gtl/components/data-table/data-table.html

```diff
@@ -24,11 +24,7 @@
       </div>
     </div>
     <div class="col-md-1 col-lg-1 col-xs-1 miq-page-counter">
-      <label>{{tableCtrl.translateString(
-        (tableCtrl.settings.startIndex + 1) +
-        ' - ' + (tableCtrl.settings.endIndex + 1) +
-        ' of ' + tableCtrl.settings.items)}}
-      </label>
+      <label>{{tableCtrl.settings.startIndex}} - {{tableCtrl.settings.endIndex}} : {{tableCtrl.settings.items}}s</label>
```

Comment:
Good notice, my `ctrl` does not work really great so hitting `ctlr` + `s` usually ends up in just random `s` in code :smile: